corehq/apps/locations/static/locations/ko/location_drilldown.async.js

```diff
@@ -107,6 +107,12 @@ function LocationModel(data, root, depth, func, withAllOption) {
   this.func = typeof func !== 'undefined' ? func : LocationModel;
   this.withAllOption = typeof withAllOption !== 'undefined' ? withAllOption : true;
 
+  this.enable = function() {
+    console.log(this.name(), this.can_edit())
```

Comment:
Expected indentation of 6 space characters but found 4. (indent)
Unexpected console statement. (no-console)
Missing semicolon. (semi)